### ISSUE TITLE
Bazarr Enhanced App stdClass::$override_url 

### ISSUE THREAD

**Issue by hoopmutter** [linuxserver/Heimdall#583]
update to latest and when try to edit app setting i get

ERROR ```
ErrorException (E_ERROR)
Undefined property: stdClass::$override_url (View: /var/www/html/app/SupportedApps/Bazarr/config.blade.php) (View: /var/www/html/app/SupportedApps/Bazarr/config.blade.php) (View: /var/www/html/app/SupportedApps/Bazarr/config.blade.php)
```

ERROR LINE: `        <?php echo Form::text('config[override_url]', (isset($item) ? $item->getconfig()->override_url : null), array('placeholder' => __('app.apps.override'), 'id' => 'override_url', 'class' => 'form-control')); ?>`

FILE: 
```
<h2><?php echo e(__('app.apps.config')); ?> (<?php echo e(__('app.optional')); ?>) <?php echo $__env->make('items.enable', \Illuminate\Support\Arr::except(get_defined_vars(), array('__data', '__path')))->render(); ?></h2>
<div class="items">
    <div class="input">
        <label><?php echo e(strtoupper(__('app.url'))); ?></label>
        <?php echo Form::text('config[override_url]', (isset($item) ? $item->getconfig()->override_url : null), array('placeholder' => __('app.apps.override'), 'id' => 'override_url', 'class' => 'form-control')); ?>
 
    </div>
    <div class="input">
        <label><?php echo e(__('app.apps.apikey')); ?></label>
        <?php echo Form::text('config[apikey]', (isset($item) ? $item->getconfig()->apikey : null), array('placeholder' => __('app.apps.apikey'), 'data-config' => 'apikey', 'class' => 'form-control config-item')); ?>
 
    </div>
    <div class="input">
        <button style="margin-top: 32px;" class="btn test" id="test_config">Test</button>
    </div>
</div>
```

**Comment by lululombard** [comment]
Same here

**Comment by KodeStar** [comment]
I think this should be fixed in 2.3.0 reopen if not
